.github/workflows/test.yml

```diff
@@ -22,10 +22,17 @@ jobs:
         run: ./scripts/install_foundry.sh
 
       - name: Run unit tests
+        id: sol_unit_tests
```

Comment:
Can you explain how this id is used?

Reply:
Was going to use this some for something, but didn't end up needing it. Removed now